cluster-autoscaler/cloudprovider/clusterapi/clusterapi_nodegroup_test.go

```diff
@@ -930,35 +945,17 @@ func TestNodeGroupDeleteNodesTwice(t *testing.T) {
 			}
 		}
 
-		// TODO(frobware) We have a flaky test here because we
-		// just called Delete and Update and the next call to
-		// controller.nodeGroups() will sometimes get stale
-		// objects from the (fakeclient) store. To fix this we
-		// should update the test machinery so that individual
-		// tests can have callbacks on Add/Update/Delete on
-		// each of the respective informers. We should then
-		// override those callbacks here in this test to add
-		// rendezvous points so that we wait until all objects
-		// have been updated before we go and get them again.
-		//
-		// Running this test with a 500ms duration I see:
-		//
-		// $ ./stress ./openshiftmachineapi.test -test.run TestNodeGroupDeleteNodesTwice -test.count 5 | ts | ts -i
-		// 00:00:05 Feb 27 14:29:36 0 runs so far, 0 failures
-		// 00:00:05 Feb 27 14:29:41 8 runs so far, 0 failures
-		// 00:00:05 Feb 27 14:29:46 16 runs so far, 0 failures
-		// 00:00:05 Feb 27 14:29:51 24 runs so far, 0 failures
-		// 00:00:05 Feb 27 14:29:56 32 runs so far, 0 failures
-		// ...
-		// 00:00:05 Feb 27 14:31:01 112 runs so far, 0 failures
-		// 00:00:05 Feb 27 14:31:06 120 runs so far, 0 failures
-		// 00:00:05 Feb 27 14:31:11 128 runs so far, 0 failures
-		// 00:00:05 Feb 27 14:31:16 136 runs so far, 0 failures
-		// 00:00:05 Feb 27 14:31:21 144 runs so far, 0 failures
-		//
-		// To make sure we don't run into any flakes in CI
-		// I've chosen to make this sleep duration 3s.
-		time.Sleep(3 * time.Second)
+		expectedSize := len(testConfig.machines) - len(testConfig.machines[7:])
```

Comment:
can we take this chance to put the magic number 7 in a constant with a comment?

Reply:
Yeah will do